rawkv.go

```diff
@@ -0,0 +1,620 @@
+// Copyright 2018 PingCAP, Inc.
```

Comment:
I think we need to distinguish raw and txn explicitly, so moving them to different directories like raw and txn is better.